app/src/main/java/com/antonioleiva/bandhookkotlin/data/Util.kt

```diff
@@ -2,4 +2,4 @@ package com.antonioleiva.bandhookkotlin.data
 
 import retrofit2.Call
 
-inline fun <T, U> Call<T>.unwrapCall(f: T.() -> U) = execute().body().f()
\ No newline at end of file
+inline fun <T, U> Call<T>.unwrapCall(f: T.() -> U) = execute().body()?.f()
```

Comment:
Retrofit 2.3.0 has marked getBody() as Nullable. Thus, DataSet class must either return a nullable list, or an empty list when body is null. In order to avoid more changes, I've chosen the second option.